util/fibers/listener_interface.cc

```diff
@@ -263,6 +273,34 @@ void ListenerInterface::Migrate(Connection* conn, fb2::ProactorBase* dest) {
   conn->OnPostMigrateThread();
 }
 
+bool ListenerInterface::SetMaxClients(uint32_t max_clients) {
```

Comment:
i think there is a contradiction in expectations here. 
this function is meant to limit number of clients, not to expand it. so reducing from UINT32_MAX to max_clients can not fail because of rlimits - we can only "improve".  Maximum we can do it to issue the warning but I do not think this function should return a status.

Reply:
changed it @romange